src/Microsoft.AspNetCore.Server.Kestrel/IKestrelServerInformation.cs

```diff
@@ -9,6 +10,15 @@ public interface IKestrelServerInformation
     {
         int ThreadCount { get; set; }
 
+        /// <summary>
+        /// The amount of time after the server begins shutting down before connections will be forcefully closed.
+        /// By default, Kestrel will wait 5 seconds for any ongoing requests to complete before terminating
```

Comment:
I picked a number that sounded good to me. I'm open to suggestions.

For most web applications that typically don't take 5 seconds or more to handle a single request, the timeout shouldn't be hit.